python/sglang/srt/layers/activation.py

```diff
@@ -165,8 +173,8 @@ def get_act_fn(
     return act_fn
 
 
-if not _is_cuda:
+if not (_is_cuda or (not _is_cuda and cpu_has_amx_support())):
```

Comment:
simplify this a little bit:
```suggestion
if not _is_cuda and not cpu_has_amx_support():
```